hazelcast/src/test/java/com/hazelcast/test/starter/HazelcastVersionLocator.java

```diff
@@ -56,21 +59,19 @@
     }
 
     public static File[] locateVersion(String version, File target, boolean enterprise) {
-        File[] files = new File[enterprise ? 6 : 3];
-        files[0] = locateMember(version, target, false);
-        files[1] = locateMemberTests(version, target, false);
-        files[2] = locateClient(version, target, false);
+        File[] files = new File[enterprise ? 4 : 2];
+        files[HAZELCAST_JAR_INDEX] = locateMainArtifact(version, target, false);
+        files[HAZELCAST_TESTS_JAR_INDEX] = locateTestsArtifact(version, target, false);
         if (enterprise) {
-            files[3] = locateMember(version, target, true);
-            files[4] = locateMemberTests(version, target, true);
-            files[5] = locateClient(version, target, true);
+            files[HAZELCAST_EE_JAR_INDEX] = locateMainArtifact(version, target, true);
+            files[HAZELCAST_EE_TESTS_JAR_INDEX] = locateTestsArtifact(version, target, true);
         }
         return files;
     }
 
     // attempts to locate member artifact in local maven repository, then downloads
-    private static File locateMember(String version, File target, boolean enterprise) {
-        File artifact = new File(LOCAL_M2_REPOSITORY_PREFIX + constructPathForMember(version, enterprise));
+    private static File locateMainArtifact(String version, File target, boolean enterprise) {
```

Comment:
Minor: since `locateMember` has been renamed to `locateMainArtifact`, maybe `downloadMember` should be renamed, too?